components/metaconfig/schema.pan

```diff
@@ -100,8 +100,6 @@ type metaconfig_textrender_convert = {
     true;
 };
 
-type caf_service_action = string with match(SELF, '^(restart|reload|stop_sleep_start)$');
```

Comment:
don't change this, list is not complete

Reply:
As discussed at workshop: It seems that ncm-metaconfig artificially restricts which CAF actions are exposed, so for now a more generic solution will require a different type definition.